resources/js/index.js

```diff
@@ -0,0 +1,49 @@
+﻿$(function () {
+  var n = 0;
```

Comment:
tìm hiểu cho anh let và var khác nhau điều gì và comment tịa đây nhé

Reply:
- var :
+ Phạm vi hàm hoặc phạm vi toàn cục
+ Có thể gán lại và có thể khai báo lại
+ Không thuộc vùng chết tạm thời - giá trị mặc định được khởi tạo là undefined

- let :
+ Phạm vi khối - khai báo bên trong hàm, biến có phạm vi hàm
+ Có thể gán lại nhưng không thể khai báo lại
+ Phụ thuộc vào vùng chết tạm thời - khi 1 biến chưa được khai báo mà sử dụng sẽ bị lỗi